dev-qt/qtnetwork/qtnetwork-5.15.9999.ebuild

```diff
@@ -1,7 +1,7 @@
 # Copyright 1999-2021 Gentoo Authors
```

Comment:
(stupid github doesn't let me add comments on lines outside the immediate context of the diff...)

Around lines 55-60 there are 3 cases of `usex` that can be turned into `usev` dropping the third (empty) argument.